patches/minecraft/net/minecraft/world/World.java.patch

```diff
@@ -541,7 +541,24 @@
              {
                  return false;
              }
-@@ -2011,6 +2163,11 @@
+@@ -1972,6 +2124,16 @@
+                 {
+                     IBlockState iblockstate1 = this.func_180495_p(blockpos$pooledmutableblockpos.func_181079_c(l3, i4, j4));
+ 
++                    Boolean result = iblockstate1.func_177230_c().isAABBInsideLiquid(this, blockpos$pooledmutableblockpos, p_72953_1_);
++
++                    if (result != null)
++                    {
++                        if (result == true)
```

Comment:
`if (result)`

Reply:
I know, but i wanted to keep it the same as that one https://github.com/MinecraftForge/MinecraftForge/blob/1.12.x/patches/minecraft/net/minecraft/world/World.java.patch#L561

Reply:
You mean #2429? I think that is just a bad practice (`result != null && result.booleanValue()` is even clearer). Back then, people rarely comment on diffs, too, and I guess it's why that patch was included.

Reply:
By the way, you should release the pooled mutable block position before returning (or jumping to the next loop if the mutable block position is retained within the loop)